### ISSUE TITLE
Form for businesses to participate

### ISSUE THREAD

**Issue by jeremiak** [sfbrigade/localfreeweb.org#1]
We need to be able to collect information from organizations that want to list their internet stations on Local Free Web. Here is the information we want to collect, note that all fields are required.
- Name (Required)
- Street Address (Required)
- City (Required)
- State (Required)
- Zipcode (Required)
- Contact Number (Required)
- URL (Required)
- Hours (Required)

**Comment by jeremiak** [comment]
Let's use Google Forms for now, and we can do whatever we want later. Here is a form I just made:
https://docs.google.com/forms/d/1wtzh1gm04XhglsC8am8e5GoTlbwvF20kOAB75cex8bY/viewform

Form action will be:
https://docs.google.com/forms/d/1wtzh1gm04XhglsC8am8e5GoTlbwvF20kOAB75cex8bY/formResponse

name - entry.2026024928
address - entry.2079212554
city - entry.389603458
state - entry.1685711416
zip - entry.1541354702
contact number - entry.1531290238
org url - entry.1499162579
hours - entry.1445978372

also make sure to include:

```
<input type="hidden" name="draftResponse" value="[,,&quot;-648267090716106434&quot;]
">
<input type="hidden" name="pageHistory" value="0">
<input type="hidden" name="fbzx" value="-648267090716106434">
```

**Comment by msecret** [comment]
Fixed with 5773cb19226d464e0e81d2ff097d98f80c1665f6
